src/App.js

```diff
@@ -1,6 +1,7 @@
-import "./App.css";
-import { Provider } from "react-redux";
-import { ThemeProvider, createTheme } from "@material-ui/core/styles";
+import './App.css';
+import { Provider } from 'react-redux';
+import { ThemeProvider, createTheme } from '@material-ui/core/styles';
+import { SnackbarProvider } from 'notistack';
```

Comment:
we have snackbar in material uI, you could have used that instead of notistack.

Reply:
Yes, but the issue was that I have to add Snackbar in each component, and in 'notistack' I just need to import function and I can invoke snack bar.